data-raw/watershed/feather/feather_river.rmd

```diff
@@ -63,44 +60,30 @@ feather_river_instream <- feather_raw %>%
          ST_spawn_wua,
          watershed)
 
-knitr::kable(align ="c", head(feather_river_instream, 5))
+knitr::kable(align ="c", head(feather_river_instream, 5), caption = "Header Descriptions: flow_cfs = flow in cubic feet per second, FR_spawn_wua = Fall Run Spawning wua, ST_fry_wua = Steelhead Fry wua, ST_juv_wua = Steelhead Juvenile wua, ST_spawn_wua = Steelhead Spawning wua")
```

Comment:
I see more things in the header description than columns in the table (ST rearing), plz double check

Reply:
fixed